### PR TITLE
✨Migrate GPT page

### LINKED ISSUE #616: 📄 Consulting Page - GPT
https://www.ssw.com.au/ssw/Consulting/GPT.aspx

### PR DESCRIPTION
**Pull request by KristenHu**
closed #616

### CODE REVIEW

**Review by johnxu16 — APPROVED** [review]
LGTM

File: content/consulting/GPT.mdx
Comment (wicksipedia) [inline content/consulting/GPT.mdx:1]: can we fix the casing on the filenames to be consistent with the other files

File: public/images/thumbs/thumb-GPT4.png
Comment (wicksipedia) [inline public/images/thumbs/thumb-GPT4.png:1]: can we fix the casing on the filenames to be consistent with the other files

File: public/images/thumbs/thumb-chatGPT.png
Comment (wicksipedia) [inline public/images/thumbs/thumb-chatGPT.png:1]: can we fix the casing on the filenames to be consistent with the other files

**Review by wicksipedia — COMMENTED** [review]
this page needs to be added to the consulting index `/consulting` (check AzDO to see how it is listed in the old website)

unfortunately, can't approve this until it builds successfully

**Review by wicksipedia — CHANGES_REQUESTED** [review]
this page needs to be added to the consulting index /consulting (check AzDO to see how it is listed in the old website)

unfortunately, can't approve this until it builds successfully

**Review by wicksipedia — APPROVED** [review]
Lgtm

**Pull request merged**